xyz-hub-service/src/main/resources/openapi-http-connector.yaml

```diff
@@ -919,6 +959,20 @@ components:
           $ref: '#/components/schemas/Context'
     Context:
       type: string
+      description: >-
+        Only applicable for composite spaces.
+        Defines the part of the composite layer on which to perform the read-action.
```

Comment:
`...composite layer...` 
We should use 'space' instead of 'layer'  is OS

Reply:
Thx. Done.